goa/Classes/menu/LevelHelpScene.h

```diff
@@ -17,6 +17,7 @@ class LevelHelpScene: public cocos2d::Node {
     static cocos2d::Scene *createScene(std::string gameName);
     static LevelHelpScene *create(std::string gameName);
 	cocos2d::ui::Button *_resumeButton;
```

Comment:
Align this line

Reply:
It is aligned, github is showing it like this.